tools/ci.sh

```diff
@@ -476,10 +481,14 @@ function ci_native_mpy_modules_build {
         arch=$1
     fi
     make -C examples/natmod/features1 ARCH=$arch
-    make -C examples/natmod/features2 ARCH=$arch
+    if [ $arch != rv32imc ]; then
+        # This requires soft-float support.
```

Comment:
Maybe make it clear that it's rv32imc that needs soft-float support, eg "This requires soft-float support on rv32imc.".

Same comment with the TLS line below, eg "This requires thread local storage support on rv32imc.".

Reply:
I found it rather clear already since the comments in question are inside an explicit rv32imc conditional block, but more clarity never hurts.  I've updated the comments as such, thanks for the suggestion.